watch-parties/src/main/webapp/videoPlayer.js

```diff
@@ -113,19 +96,45 @@ function longPolling() {
         success: function(updater){
             if(!host){
                 updatePlayer(updater.status);
+                checkDifference(updater.timeStamp);
             }
         },
         error: function(err) {
-            console.log("Error");
+            console.log("Error in longPolling() occured!");
```

Comment:
Use `console.error` and provide the `err` object as a second argument.
```suggestion
            console.error("Error in longPolling() occured!", err);
```
The `console.*` methods can print complex objects (such as an error object) to the console so that you can explore them. Error objects also commonly provide stack traces to help you debug the issue.

`console.log`
![image](https://user-images.githubusercontent.com/1214564/88441883-6252c900-cdc7-11ea-897e-544d3ca6567b.png)
vs `console.error`
![image](https://user-images.githubusercontent.com/1214564/88441887-654db980-cdc7-11ea-97ac-ba2cb3e65996.png)